docs/content/users/quickstart.md

```diff
@@ -318,6 +318,17 @@ In the web browser, log into your account using `admin` and `publish`.
 
 Visit [Ibexa documentation](https://doc.ibexa.co/en/latest/getting_started/install_with_ddev/) for more cases.
 
+## Joomla
+
+```bash
+mkdir my-joomla-site && cd my-joomla-site
+tag=$(curl -L "https://api.github.com/repos/joomla/joomla-cms/releases/latest" | docker run -i --rm ddev/ddev-utilities jq -r .tag_name) && curl -L "https://github.com/joomla/joomla-cms/releases/download/$tag/Joomla_$tag-Stable-Full_Package.zip" -o joomla.zip
+unzip ./joomla.zip && rm joomla.zip
+ddev config --project-type=php --webserver-type=apache-fpm --upload-dirs=images
+ddev start
+ddev launch
+```
+
```

Comment:
In the "Database Configuration" section, 
* "host name" is `db`
* "database username" is `db`
* "database password" is `db`
* "database name" is `db`

Follow the instructions to delete the text file that the Joomla! installation creates and continue installation.

Reply:
good point in regards of the hand holding. on that db config page i would been uncertain as well. pushed up a suggestion ready to review.

Reply:
Joomla can be installed by CLI. Would it be better if we add here the command to perform a CLI installation?

Reply:
OH that is possible?! i was completely unaware of that. already found the docs. on it, working on a draft for that install command.

Reply:
@carcam how about: 

```ddev php installation/joomla.php install --site-name="My Joomla Site" --admin-user="John Doe" --admin-username=admin --admin-password=AdminAdmin1! --admin-email=admin@mail.com --db-host=db --db-user=db --db-pass=db --db-name=db```

the only downside is that there is still a confirmation prompt for the database type, the db prefix, the encryption for the db connection and the question of relative or absolute path to the public folder. two questions. 

should all those four be set in that command? and what would be the best approach for the public folder, relative or absolute?

Reply:
cuz somehow i am unable to find docs for the 5.x cli install, i am only able to find 4.x ( https://docs.joomla.org/Special:MyLanguage/J4.x:Joomla_CLI_Installation ) and that is missing the public folder setting

Reply:
There is a great article with a complete example at Akeeba's blog: 

https://www.akeeba.com/documentation/admin-tools-joomla/custom-public.html

To be more specific:

> there is still a confirmation prompt for the database type, the db prefix, the encryption for the db connection

As per the example you may use include them in the parameters:

`--db-type=mysql --db-encryption=0 --db-host=db --db-user=db --db-pass="db" --db-name=db --db-prefix=ddev_`

I have used *ddev* as prefix for simplicity, but you may set a random word or whatever you want.

> or the public folder, relative or absolute?

Public folder is a feature meant for experts so I do not think we should use the conventional way which would be using the folder that contains all the files. So:

`--public-folder=/var/www/html`

And that should get rid of all the questions ;)

Reply:
ahhhhhh public and private is setting the docroot. and there is no option to simply go with the defaults if an option is not set. got it. 

i've added the points you suggested. but for ```ddev php installation/joomla.php install --site-name="My Joomla Site" --admin-user="John Doe" --admin-username=admin --admin-password=AdminAdmin1! --admin-email=admin@mail.com --db-type=mysql --db-encryption=0 --db-host=db --db-user=db --db-pass="db" --db-name=db --db-prefix=ddev_ --public-folder=/var/www/html```
i get the following error: 
```
Install Joomla
==============

Checking system requirements...OK
Collecting configuration...OK
Validating DB connection...OK
Creating and populating the database...OK
Writing configuration.php and additional setup ...OK
Deleting /installation folder...OK
Creating the public folder...

                                                                                                                        
 [ERROR] Unable to create the given folder, index.php already exists.                                                   
                                                                                                                        

Failed to run php installation/joomla.php install --site-name=My Joomla Site --admin-user=John Doe --admin-username=admin --admin-password=AdminAdmin1! --admin-email=admin@mail.com --db-type=mysql --db-encryption=0 --db-host=db --db-user=db --db-pass=db --db-name=db --db-prefix=ddev_ --public-folder=/var/www/html: exit status 1
```
the path already exists i suppose.

Reply:
ok pushed a working commit. @rfay and @stasadev  i think joomla is the first quickstart that has an actual user name to be added. i went with John Doe and the password has a lot of requirements so i went with AdminAdmin1! and for the email i went with admin@mail.comas the placeholder. are those ok or should we agree on a more generic name? 

and aside the cli install i also pointed the launch command to the admin ui login page instead of the home page now.

Reply:
and strange. the last commit is listed https://github.com/ddev/ddev/pull/6255/commits/8e413040811c8d83419f54b0016e587a024175c2 the checks are all shown as passed but if i go to the preview i still get the old?!

Reply:
It's correct now.

Reply:
yep excellent. thanks for checking. and i just noticed taking a look at the preview that the explanations after the install steps are obsolete now with the cli install method. i push another commit to remove the following again ```Now follow the instructions in the installer. The only detail to note, within the Database Configuration section, for the database type use the default MySQLi, and for the host name, the database user name, the database password, and the database name use db. The database prefix and the connection encryption settings can be left at the default settings.```